src/navigation/_CollapsingNavigationMenu.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/role-supports-aria-props */
```

Comment:
This disable line is to deal with an `aria-haspopup is not supported by the role link` linting error.

Reply:
this should be address here: https://github.com/cerner/terra-dev-site/pull/279